src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -451,21 +455,19 @@ export default {
         this.$refs.entryInput.scrollIntoView({ behavior: 'smooth', block: 'nearest' });
       }
     },
-    validateLink(originalLinkRenderer, href, title, text) {
+    validateLink(options) {
+      const { href, text } = options;
       try {
         const domain = new URL(href).hostname;
         const urlIsWhitelisted = this.urlWhitelist.some((partialDomain) => {
           return domain.endsWith(partialDomain);
         });
+
         if (!urlIsWhitelisted) {
           return text;
         }
-        const linkHtml = originalLinkRenderer.call(this.renderer, href, title, text);
-        const linkHtmlWithTarget = linkHtml.replace(
-          /^<a /,
-          '<a class="c-hyperlink" target="_blank"'
-        );
-        return linkHtmlWithTarget;
+
+        return `<a class="c-hyperlink" target="_blank", href="${href}">${text}</a>`;
```

Comment:
Is this `,` accidentally in the url or is that some part of marked?

Reply:
That's uh....that's a typo. Good catch!